bookshelves.md

```diff
@@ -8,39 +8,62 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
 
 “Bookshelves” are lists of PG books that help users navigate PG. See them [here](https://www.gutenberg.org/wiki/Category:Bookshelf).
 
-
 ## Problem
 
 - System for creating and updating bookshelves is failing
-    - PHP/wiki interface
-    - Loaded into production Postgres db by scraping wiki!
-    - Few people maintaining it.
-    - No way to create personal bookshelves or reading lists
-    - Recurring security issues with PHP
-
+  - PHP/wiki interface
+  - Loaded into production Postgres db by scraping wiki!
+  - Few people maintaining it.
+  - No way to create personal bookshelves or reading lists
+  - Recurring security issues with PHP
 
 ## Proposed Solution
 
 - Registrationless Bookshelf Builder
-    - HTML5 local storage + javascript
-    - Button on Item page
-    - Import/export lists for sharing
-
-- Bookshelf Management App
-    - Python/Flask or Python/CherryPy with Postgres backend
-    - Managers can make new bookshelves and categories
-    - Moderators can add/remove items from Builder Bookshelves
 
+  - HTML5 local storage + javascript
+  - Button on Item page
+  - Import/export lists for sharing
 
+- Bookshelf Management App
+  - Python/Flask or Python/CherryPy with Postgres backend
+  - Managers can make new bookshelves and categories
+  - Moderators can add/remove items from Builder Bookshelves
 
 ## Goal
 
 Have prototype system running by April, 2021 for public review, move to production bin May.
 
+## Timeline
+
+| Dates               |                                                               Milestones                                                               |
+| ------------------- | :------------------------------------------------------------------------------------------------------------------------------------: |
+| 09/14/20 - 9/21/20  | Review existing code/documentation, and implementation goals for deeper understanding of technical aspects of problem |
+| 09/21/20 - 12/02/20  | Document requirements of PG deployment environment and evaluate technology options. |
+| 10/02/20            |   Deliver user stories for dev team to iteratively implement; allowing the dev team to reevaluate any cases.   |
+| 10/09/20            |   Deliver implementation plan detailing technical feasability and decisions. Document what tools and technologies will be used.   |
+| 10/12/20 - 03/19/21 |    Main Development Period: Iteratively develop user stories several sprints. Demo new features at the end of sprints for feedback.    |
+| 03/19/21            |                                                       Final demo of new features                                                       |
+| 03/19/21 - 04/30/21 |                              QA period to resolve issues and bugs. Final preparation for product release                               |
+
+_Dates/Milestones are tentative and subject to change._
+
 ## Team
 
-Add names and links here.
+Adam Undus (aundus@stevens.edu)
+Liam Nagel (nagel@stevens.edu)
```

Comment:
Your invitation to nagel@stevens.edu has bounced.

Reply:
Made a typo in the email fix here: https://github.com/EbookFoundation/capstone-projects/pull/10
I resent an invite to him.